event_sale_registration_multi_qty/models/sale_order.py

```diff
@@ -11,15 +10,17 @@ class SaleOrderLine(models.Model):
     _inherit = 'sale.order.line'
 
     @api.multi
-    def _update_registrations(self, confirm=True, registration_data=None):
+    def _update_registrations(self, confirm=True, cancel_to_draft=False,
+                              registration_data=None):
         """ Writes registrations """
         Registration = self.env['event.registration']
         registrations = Registration.search([
             ('sale_order_line_id', 'in', self.ids)
         ])
         for so_line in self.filtered('event_id'):
             if not so_line.event_id.registration_multi_qty:
-                super(SaleOrderLine, so_line)._update_registrations()
+                super(SaleOrderLine, so_line)._update_registrations(
+                    confirm, cancel_to_draft, registration_data)
```

Comment:
Please respect keyword arguments putting them as so in the call (`confirm=confirm, cancel_to_draft=...`).